src/test/ensemble_metadata_test.cpp

```diff
@@ -83,20 +83,23 @@ TEST(EnsembleMetadata, MultipleNodesOnDifferentLevelsUsingTheSamePipelineInputs)
     ModelConfig sum_model_config = SUM_MODEL_CONFIG;
     ASSERT_EQ(manager.reloadModelWithVersions(sum_model_config), StatusCode::OK);
 
+    const std::string& INCREMENT_MODEL_INPUT_NAME = DUMMY_MODEL_INPUT_NAME;
```

Comment:
Aren't those defined in test_utils.hpp?

Reply:
I rename the variable to better match its purpose in this specific unit test.